### ISSUE TITLE
URL: Update Work Manager URL

### ISSUE THREAD

**Issue by acshields** [topcoder-platform/work-manager#1121]
Update Work Manager application's URL to be: connect.topcoder.com/workmanager

**Comment by vikasrohit** [comment]
@acshields I am not sure if we can really call it bug fix or quick win because technically it is major change and we might have to do a good regression testing after such changes. fyi @lakshmiathreya

**Comment by acshields** [comment]
@vikasrohit - good to know. I will add this to a larger shape-up pitch then & combine efforts with Will & TaaS App so we have a standard URL naming convention across the client experience. Thanks!

**Comment by vikasrohit** [comment]
Yep, the dev efforts might not be large but we need full cycle of QA regression to avoid any dangling link.

**Comment by acshields** [comment]
@maxceem - makes sense! probably a lot of places we don't even realize that are linked to the URL.
